src/components/Dashboard.jsx

```diff
@@ -25,18 +25,6 @@ class Dashboard extends Component {
     }
     this.props.fetchCampaignData(this.props.admin);
   }
-  getMediaUrl() {
```

Comment:
Why are we removing this?

Reply:
It is no longer passed as props in this card.